### ISSUE TITLE
[ALM Accelerator - BUG]: Cannot read properties of null (reading id)

### ISSUE THREAD

**Issue by linujohn22** [microsoft/coe-starter-kit#3516]
### Describe the issue

On commit operations  we are getting below error . 

![image](https://user-images.githubusercontent.com/44782172/186647753-cc185851-0e72-4304-af0c-ee0e09c2ead8.png)

### Expected Behavior

it should work fine

### What component are you experiencing the issue with?

ALM Accelerator Pipelines

### What solution version are you using?

20220726

### Steps To Reproduce

Click on commit button

**Comment by mikefactorial** [comment]
@linujohn22 thanks for reporting this we're aware of the issue and are looking into pushing out a fix as soon as possible. Will report back when we have resolved the issue

**Comment by mikefactorial** [comment]
Duplicate of #3515

**Comment by linujohn22** [comment]
We have installed ALM accelerator first week of Aug 2022 . We were able to successfully deploy some sample solutions . We showed demo to client as well.  Now we started deploying actual solutions and started getting this error . Can you please explain how it was working earlier and it stopped working now ? we have to give some explanation to client .

**Comment by linujohn22** [comment]
> @mikefactorial We have installed ALM accelerator first week of Aug 2022 . We were able to successfully deploy some sample solutions . We showed demo to client as well. Now we started deploying actual solutions and started getting this error . Can you please explain how it was working earlier and it stopped working now ? we have to give some explanation to client .

@mikefactorial We have installed ALM accelerator first week of Aug 2022 . We were able to successfully deploy some sample solutions . We showed demo to client as well. Now we started deploying actual solutions and started getting this error . Can you please explain how it was working earlier and it stopped working now ? we have to give some explanation to client .

**Comment by mikefactorial** [comment]
This is a bug that was introduced in our preparations for the September release. We've discussed the issue internally related to how we are distributing the coe-cli as part of our pipelines and are working on a solution to avoid this issue in the future. A fix has been released for the immediate issue and you shouldn't be seeing the failure in your pipelines now. We're hoping to have the long term solution as soon as possible.
